src/objects/Scores.ts

```diff
@@ -12,14 +12,15 @@ export class Scores extends Phaser.GameObjects.Container {
 
     public ShowScore() {
         this.scoreText = this.snakeScene.add.text(16, 514, 'Score: ' + this.score, { fontSize: '32px', fill: '#000' });
+
     };
     public addPoint() {
         this.score++;
-        this.snakeScene.Ncapsule--;
+        this.snakeScene.Nhashtags--;
         this.scoreText.setText('Score: ' + this.score)
     };
     public BonusScore() {
-        if (this.score === 1) {
+        if (this.score >= 1 && this.score < 5) {
```

Comment:
tu utilises la comparaison pas mal de fois, tu peux faire une méthode scoreIsBetween(1, 5) et ça te fait la comparaison. Pareil pour la condition if (this.snakeScene.timerEvent.delay !== this.snake.....